src/get-started/beam-overview.md

```diff
@@ -37,6 +37,7 @@ Beam currently supports Runners that work with the following distributed process
 * Apache Gearpump (incubating) ![Apache Gearpump logo]({{ "/images/logos/runners/gearpump.png" | prepend: site.baseurl }})
 * Apache Spark ![Apache Spark logo]({{ "/images/logos/runners/spark.png" | prepend: site.baseurl }})
 * Google Cloud Dataflow ![Google Cloud Dataflow logo]({{ "/images/logos/runners/dataflow.png" | prepend: site.baseurl }})
+* Apache Samza ![Apache Samza logo]({{ "/images/logos/runners/samza.png" | prepend: site.baseurl }})
```

Comment:
If you check out the generated page, this logo really is huge! It might be easy to style it in markdown, or find a smaller image.